check.go

```diff
@@ -234,14 +238,14 @@ func (c *CheckRunner) UpdateChecks(checks api.HealthChecks) {
 		} else if definition.TCP != "" {
 			anyUpdates = c.updateCheckTCP(check, checkHash, &definition, updated, added)
 		} else {
-			c.logger.Printf("[WARN] check %q is not a valid HTTP or TCP check", checkHash)
+			c.logger.Warn("", "check ", hclog.Fmt("%q is not a valid HTTP or TCP check", checkHash))
 			continue
 		}
 
 		// if we had to fix the interval and we had to update the service, put some trace out
 		unmodifiedDef := check.Definition
 		if anyUpdates && unmodifiedDef.IntervalDuration < c.MinimumInterval {
-			c.logger.Printf("[WARN] Check interval too low at %v for check %s", unmodifiedDef.Interval, check.Name)
+			c.logger.Warn("Check interval too low ", "Interval", hclog.Fmt("%v for check %s", unmodifiedDef.Interval, check.Name))
```

Comment:
This could be reworded to also use the k/v pairs. I suggested lower-casing "Interval" since in the hclog documentation it looks like they use camel-case for keys
```
c.logger.Warn("Check interval too low", "interval", hclog.Fmt("%v", unmodifiedDef.Interval), "checkName", check.Name)
```